Apps/Common/src/Services/EmailQueueService.cs

```diff
@@ -33,11 +33,11 @@ namespace HealthGateway.Common.Services
     /// </summary>
     public class EmailQueueService : IEmailQueueService
     {
-        private const int VerificationExpiryDays = 5;
 #pragma warning disable SA1310 // Disable _ in variable name
         private const string INVITE_KEY_VARIABLE = "InviteKey";
         private const string ACTIVATION_HOST_VARIABLE = "ActivationHost";
         private const string ENVIRONMENT_VARIABLE = "Environment";
+        private const string VERIFICATION_EXPIRY_HOURS = "VerificationExpiryHours";
```

Comment:
please update it to "ExpiryHours", which i used for the template.

Reply:
ExpiryHours doesn't have any meeaning in the WebConfig, as it can have too many meaning such as Expiry SMS, Email, etc... Email template may have enought context to understand ExpiryHours but not in WebConfig. Ideally, we should use the same name everywhere as it is hard to maintain software with different names for the same parameter.